vgpu-manager/ubuntu20.04/nvidia-driver

```diff
@@ -111,6 +111,20 @@ _create_dev_char_directory() {
     fi
 }
 
+_set_fw_search_path() {
+    local nv_fw_search_path="$RUN_DIR/driver/lib/firmware"
+    local fw_path_config_file="/sys/module/firmware_class/parameters/path"
```

Comment:
Can we check if the file exists, this may cause an error when this build is used with older gpu-operator versions where sysfs path is not mounted.

Reply:
Note, we already mount the host /sys in the vGPU Manager container, see: https://github.com/NVIDIA/gpu-operator/blob/master/assets/state-vgpu-manager/0500_daemonset.yaml#L61-L62 So no GPU Operator changes are required and thus this change works with existing operator versions.

Reply:
@shivamerla does my comment address your concern? Or would you still like to see a file existence check added here before we merge this.

Reply:
yes, if `/sys` was mounted all along then this is fine.